persim/images.py

```diff
@@ -2,29 +2,37 @@
 from itertools import product
 import collections
 
+import copy
 import numpy as np
-from scipy.stats import multivariate_normal as mvn
 from scipy.stats import norm
 import scipy.spatial as spatial
 import matplotlib.pyplot as plt
-
+from persim import images_kernels
+from persim import images_weights
+from joblib import Parallel, delayed
```

Comment:
Could you add `joblib` and `deprecated` to the `setup.py` requirements so they get installed correctly.

Reply:
Done.